Decred Wallet/Features/History/TransactionTableViewCell.swift

```diff
@@ -56,30 +43,29 @@ class TransactionTableViewCell: BaseTableViewCell {
             dateformater.string(from: Date2 as Date)
             
             self.dateT.text = dateformater.string(from: Date2 as Date)
-            let amount = Decimal(transaction.Amount / 100000000.00) as NSDecimalNumber
             let requireConfirmation = Settings.spendUnconfirmed ? 0 : 2
             
             if (transaction.Type.lowercased() == "regular") {
                 if (transaction.Direction == 0) {
```

Comment:
Can we have an enum that defines what zero is in this `transaction.Direction == 0` to something like `transaction.Direction == left`

Reply:
Oops, this should actually use dcrlibwallet constants. Will correct.